### ISSUE TITLE
exampleCourse has incorrect GitHub links on questions settings pages

### ISSUE THREAD

**Issue by echuber2** [PrairieLearn/PrairieLearn#5025]
It looks like whatever formula is used to generate the GitHub links on question settings pages is not working correctly for the exampleCourse that sits in the main PL repo.

Example:
https://www.prairielearn.org/pl/course/108/question/4942648/settings
The "view on GitHub" link leads to here:
https://github.com/PrairieLearn/PrairieLearn/tree/master/questions/demo/drawing/graphs
but it should lead to here:
https://github.com/PrairieLearn/PrairieLearn/tree/master/exampleCourse/questions/demo/drawing/graphs

**Comment by nwalters512** [comment]
Good catch - it looks like the bit of logic that generates that code doesn't take into account the fact that the example course is a subdirectory of a repo, whereas we assume every other course is at the repo root. We special case the example course in many other places throughout PL; we'll have to do the same here too.

**Comment by echuber2** [comment]
Perhaps this is a difficult suggestion if it's already being special-cased in many places, but would it be better if each course had a setting for root directory? That would at least generalize the problem.

**Comment by trombonekenny** [comment]
Is this resolved? Perhaps by #5288?

**Comment by nwalters512** [comment]
Yep, I think we can close this!
